queue_services/entity-emailer/src/entity_emailer/email_processors/continuation_out_notification.py

```diff
@@ -0,0 +1,136 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Email processing rules and actions for Continuation Out notifications."""
+from __future__ import annotations
+
+import base64
+import re
+from http import HTTPStatus
+from pathlib import Path
+
+import requests
+from entity_queue_common.service_utils import logger
+from flask import current_app
+from jinja2 import Template
+from legal_api.models import Business, Filing, UserRoles
+
+from entity_emailer.email_processors import get_filing_info, get_recipient_from_auth, substitute_template_parts
+
+
+def _get_pdfs(
+        status: str,
+        token: str,
+        business: dict,
+        filing: Filing,
+        filing_date_time: str,
+        effective_date: str) -> list:
+    # pylint: disable=too-many-locals, too-many-branches, too-many-statements, too-many-arguments
+    """Get the pdfs for the Continuation Out output."""
+    pdfs = []
+    attach_order = 1
+    headers = {
+        'Accept': 'application/pdf',
+        'Authorization': f'Bearer {token}'
+    }
+
+    if status == Filing.Status.COMPLETED.value:
```

Comment:
Can remove this `if` condition since message tracker will skip PAID status

Reply:
The only reason I left that in was in case this code was run with a filing that was not COMPLETED, as a safety check. I left it in for CCO, too.

Reply:
If this code run with PAID status, then it will fail before executing get_pdf (here: https://github.com/bcgov/lear/pull/2071/files#diff-58c9c91cac9d208ab7dd407c5ac8f11a99d3b6eb23ea3ced41e8c3f488479dc9R89). So you can actually remove from both CO and CCO